ykdl/extractors/zhuafan.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import base64
+import json
+
+from ykdl.videoinfo import VideoInfo
+from ykdl.extractor import VideoExtractor
+from ykdl.util.match import match1
+from ykdl.util.html import get_content
+
+
+class ZHULive(VideoExtractor):
+    name = u"Zhuafan"
+
+    def decodeencoded(self, playinfo):
```

Comment:
This is not compatible with py2, can you fixed it, or let me do it?

Reply:
I will leave that for you, as it was time consuming to work with their flash player already